src/core/api.ts

```diff
@@ -88,10 +88,10 @@ class API {
    * @param args An array of arguments to pass as Textile args headers
    * @param opts An object of options to pass as Textile options headers
    */
-  protected async sendGatewayGet(path: string, args?: string[], opts?: KeyValue, headers?: KeyValue) {
+  protected async sendGatewayGet(path: string, headers?: KeyValue) {
```

Comment:
What is this method used for?

Reply:
nothing anymore. but i realized we could expose a method for gateway requests so that users don't have to manage their own wrappers where we already have them.